lib/generator/exercise_cases.rb

```diff
@@ -25,6 +25,23 @@ def indent_lines(code, depth, separator = "\n")
     code.join(separator + ' ' * depth)
   end
 
+  # use to indent multi line workloads with blank
+  # lines
+  #   indent_text(4, lines.join("\n"))
+  def indent_text(depth, text)
+    text.lines.reduce do |obj, line|
+      obj << (line == "\n" ? line : ' ' * depth + line)
+    end
+  end
```

Comment:
I'd like to see the `exercise_cases` changes in a separate PR and have this one be just about moving the template files.